kde-plasma/plasma-pa/plasma-pa-9999.ebuild

```diff
@@ -33,6 +33,7 @@ DEPEND="
 	>=kde-plasma/libplasma-${PVCUT}:6
 	media-libs/libcanberra
 	media-libs/libpulse
+	media-libs/pulseaudio-qt
```

Comment:
We don't need a slot op?

Reply:
I'll add that, but it does require newer than 1.4.0, should I put > 1.4.99 or something like that?

Reply:
`PROJECT_VERSION` in upstream CMakeLists.txt is already bumped to `1.5.0` so use that.

Depending on `${KFMIN}:6` does not work at all right now as @jospezial already pointed out.

Reply:
https://invent.kde.org/plasma/plasma-pa/-/merge_requests/254